frontend/src/metabase/visualizations/components/settings/ChartSettingColumnEditor.tsx

```diff
@@ -0,0 +1,204 @@
+/* eslint-disable react/prop-types */
```

Comment:
In .tsx files, I think prop-types are not required, so we don't need this eslint disabler.

Reply:
Ohhh, good catch. I started with some copy paste imports and this hitched a ride haha